invenio_rdm_records/config.py

```diff
@@ -338,6 +338,24 @@ def make_doi(prefix, record):
 in DataCite XML format.
 """
 
+RDM_CUSTOM_FIELDS_NAMESPACES = {}
```

Comment:
I would call it just `RDM_NAMESPACES` so it can be reused across the system and simplify the name...